src/MediaWiki/Search/ProfileForm/Forms/CustomForm.php

```diff
@@ -122,10 +122,10 @@ public function makeFields( $definition ) {
 
 			// Find request related value for the active form
 			if ( $this->isActiveForm ) {
-				$vals = $this->request->getArray( $name );
+				$vals = $this->request->getArray( $name, [] );
 
 				$i = $this->fieldCounter[$name];
-				$value = isset( $vals[$i] ) ? $vals[$i] : $vals[0];
+				$value = $vals[$i] ?? null;
```

Comment:
I'm not sure if this should be '' instead of null.

Reply:
Should be just `''` because it was the previous value on line 106

Reply:
Thanks, done.